completions/_nox

```diff
@@ -0,0 +1,14 @@
+# 3rd party completion loader for argcomplete commands     -*- shell-script -*-
+# sourced using no args to `register-python-argcomplete`.
+#
+# This serves as a fallback in case the completion is not installed otherwise.
+
+eval -- "$(
+    PATH=$(type -p "$1" 2>/dev/null | command sed 's,/[^/]*$,,')${PATH:+:$PATH}
```

Comment:
@scop I'm sorry, I now looked at it again and noticed that this treatment has a problem when `type -p "$1"` failed. The result of `$(...)` becomes empty, and the PATH environment becomes `PATH=:<original content of PATH>`. The empty path in `PATH` results in the execution of an executable file in the current directory, which can be a vulnerability.

```suggestion
    bin_path=$(type -p "$1" 2>/dev/null | command sed 's,/[^/]*$,,')
    [[ $bin_path ]] && PATH=$bin_path${PATH:+:$PATH}
```

Reply:
Yeah, I observed that as well in https://github.com/scop/bash-completion/pull/917#discussion_r1161092202 above, but thought that wouldn't be an issue and that `.` in `$PATH` would be required for current dir to be considered, not with an empty segment. But you seem right indeed :/ Anyway this already went in, would you mind submitting another PR (pre-approved, feel free to merge directly) for this?

Reply:
> Anyway this already went in, would you mind submitting another PR (pre-approved, feel free to merge directly) for this?

Thank you for your reply. I'll open a PR.

Reply:
#920